cmd/cdi-controller/controller.go

```diff
@@ -113,7 +115,14 @@ func start(cfg *rest.Config, stopCh <-chan struct{}) {
 		klog.Fatalf("Error building example clientset: %s", err.Error())
```

Comment:
This is not your issue, but its funny no one noticed the 'example' part of the message.